src/bugreporter/odemis_bugreporter.py

```diff
@@ -318,6 +318,23 @@ def compress_files(self):
             if delphi_calib_reps and (time.time() - os.path.getmtime(delphi_calib_reps[-1])) / 3600 < 24:
                 files.append(delphi_calib_reps[-1])
 
+            # Add the latest fastem-calibration image if it's possibly related (ie, less than a week old)
+            fastem_calib_dir = os.path.join(home_dir, "development/fastem-calibrations/images/cell-translation")
```

Comment:
Does this part have to be updated given this change https://bitbucket.org/delmic/fastem-calibrations/src/ec4b208ca92c5c8f4166b3db280d20b7cce4d82b/src/fastem_calibrations/util.py#lines-65?

Reply:
good catch, fixed it.